File: geohelp-organizer-app/src/app/api/approve-emergency-request.ts
export class ApproveEmergencyRequest {

  public eventID: number;
  public passphrase: string;
  public emergencyID: number;
  public message: string;
  public approved: boolean;


  constructor(eventID: number, passphrase: string, emergencyID: number, message: string, approved: boolean) {
    this.eventID = eventID;
    this.passphrase = passphrase;
    this.emergencyID = emergencyID;
    this.message = message;
    this.approved = approved;
  }

}

File: geohelp-backend/src/main/java/ch/geohelp/geohelpbackend/api/model/ApproveEmergencyRequest.java
package ch.geohelp.geohelpbackend.api.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter


public class ApproveEmergencyRequest {
    long eventID;
    String passphrase;
    long emergencyID;
    String message;
    boolean approved;
}
